lb/lb_test.go

```
@@ -0,0 +1,107 @@
+	exec_order := []int{}
+	b1 := &TestBackend{idx: 0, healthy: true, onHandleRequest: func() {
+		exec_order = append(exec_order, 0)
+	}}
+	b2 := &TestBackend{idx: 1, healthy: true, onHandleRequest: func() {
+		exec_order = append(exec_order, 1)
+	}}
+	lb := lb.NewLoadBalancer(":8080", []lb.Backend{b1, b2})
+
+	fmt.Println("starting client")
+	defer lb.Close()
+	go lb.Start()
```

golangci-lint (1.62.2)

Error return value of `lb.Start` is not checked

(errcheck)